src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{4f3766dd-5201-42c0-aa83-41e71f56c7a0}",
+    "id": "1011"
+  },
+  "description": {
+    "text": "A project template for creating an ASP.NET Core SignalR real-time application with example SignalR Hubs.",
+    "package": "{4f3766dd-5201-42c0-aa83-41e71f56c7a0}",
+    "id": "1012"
+  },
+  "order": 100,
```

Comment:
Order matters. Where does this show up in our list at the moment?

Reply:
I think @mkArtakMSFT is in charge of the template order.

Reply:
changed it to 450 so it'd be later in the list but before gRPC.